configure-buildfarm/tasks/main.yml

```diff
@@ -38,13 +38,12 @@
 
 - pause:
     prompt: |
-         The BuildFarm was successfully installed, Do you want to configure your existing RHMAP installation to use the new BuildFarm ?
-
-         If you want to continue please add your current ~/.ssh/id_rsa.pub key here
-
+         Jenkins was successfully installed. In order to configure Jenkins and continue with the installation please add your current public key (~/.ssh/id_rsa.pub) in the input box 'SSH Public Keys' at the link below:
```

Comment:
"Note: The following step is required. Playbook run will fail if skipped." + following text. reason: you want user to be advised first